tests/MongoDB.EntityFrameworkCore.UnitTests/ValueGeneration/MongoValueGeneratorSelectorTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿/* Copyright 2023-present MongoDB Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.ValueGeneration;
+using MongoDB.Bson;
+using MongoDB.EntityFrameworkCore.ValueGeneration;
+
+namespace MongoDB.EntityFrameworkCore.UnitTests.ValueGeneration;
+
+public class MongoValueGeneratorSelectorTests
+{
+    private readonly IModel __model;
```

Comment:
Just curious what double-underscores means here?

Reply:
Double-underscore is private static which comes from the naming rules in the .editorconfig we borrowed from the Mongo C# Driver repository. Here however their name should have been corrected when they stopped being static, will fix.